cms/jinja2/templates/pages/statistical_article_page.html

```diff
@@ -83,13 +83,17 @@ <h1 class="ons-u-fs-3xl common-header__heading">
             {% call onsDetails({"id": "corrections-and-notices", "title": corrections_and_notices_title}) %}
                 {% if page.corrections %}
                     <h3>{{ _("Corrections") }}</h3>
-                    {% for correction in page.corrections %}
-                        {% include_block correction %}
-                    {% endfor %}
+                    <div id="corrections">
+                        {% for correction in page.corrections %}
+                            {% include_block correction %}
+                        {% endfor %}
+                    </div>
                 {% endif %}
                 {% if page.notices %}
                     <h3>{{ _("Notices") }}</h3>
-                    {{ page.notices }}
+                    <div id="notices">
+                        {{ page.notices }}
+                    </div>
```

Comment:
random thought: considering the article body has sections with editorial titles which the get slugified and used as IDs for the table of contents, there is a chance that one of the sections may be "Notices" so we'd have clashing IDs. So, a) could this be more general b) is it needed at all? Or are you pre-empting the DS component?

Reply:
To be honest I only did this to simplify the locator in the test:

https://github.com/ONSdigital/dis-wagtail/blob/7925f572d9e1c4080b26bb312a998cf794356b38/functional_tests/steps/statistical_article_page.py#L178-L180

...but I see your point. I could either prefix the ID with something like `"cn_"` or rework the locator so that the id here can be removed. What do you think?

Reply:
let's leave as is for now